packages/fx-core/src/plugins/resource/function/resources/message.ts

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { FunctionPluginPathInfo as PathInfo } from "../constants";
+import * as path from "path";
+import { FunctionPluginInfo, FunctionPluginPathInfo as PathInfo } from "../constants";
+
+export function normalizeMessage(message: string): string {
```

Comment:
We'd better not add customized format in our plugin unless we synced with solution.

Reply:
Good point. Removed the format for notification messages.